ui/src/containers/appView/SaveAppModalContainer.js

```diff
@@ -6,6 +6,15 @@ import { connect } from 'react-redux';
 import { toggleSaveAppModal } from '../../actions/actions-ui';
 import ConfirmationModal from '../../components/common/ConfirmationModal';
 
+/**
+ * Redux container for the Save App Modal
+ *
+ * @param {Boolean} isOpen - The open/closed state of the modal
```

Comment:
Just noticed this as I am going through the same thing. This comment is technically not correct as all params are under the one `props` param. This can be specified like this.

```
/**
 * Form for user login
 * @param {object} props Component props
 * @param {Boolean} props.isOpen - The open/closed state of the modal
 */
```
the same for the rest